src/shogun/mathematics/linalg/LinalgNamespace.h

```diff
@@ -125,10 +135,7 @@ SGVector<T> from_gpu(const SGVector<T>& vector)
 		return SGVector<T>(data, vector.vlen);
 	}
 	else
-	{
-		SG_SWARNING("Trying to run GPU code without GPU backend registered.\n");
-		return vector;
-	}
+		SG_SERROR("No GPU backend registered.\n");
```

Comment:
from_gpu should still work without gpu backend!
So that we can run GPU code without a GPU, important!